Libplanet/Blockchain/BlockChain.cs

```diff
@@ -62,6 +62,11 @@ public class BlockChain<T>
         /// </summary>
         private IDictionary<TxId, Transaction<T>> _transactions;
 
+        /// <summary>
+        /// Cached genesis block.
+        /// </summary>
+        private Block<T> _genesis;
```

Comment:
Shouldn't `_genesis` be set?

Reply:
It will be set only when `BlockChain<T>.Genesis` called.

Reply:
Oh, I mistaken `??=` as `??`.